src/System.Net.Utilities/src/System/Net/NetworkInformation/Icmpv4MessageConstants.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    internal static class Icmpv4MessageConstants
```

Comment:
Naming nit: I think the v in v4 should be capitalized, to take it out of the ICMP abbreviation.